### PR TITLE
redirect then 200 is success

### PR DESCRIPTION
**Pull request by GarrettArm**
Continuing pull request from https://github.com/Islandora/islandora_paged_content/pull/147

New changes:  "availible" --> "available"
                         strpos instead of strstr, for needle/haystack
                         removed commented-out code

Old changes:  allows 302 redirect then a 200 to pass as a success.  See above link for more info

### CODE REVIEW

**Comment by GarrettArm** [comment]
@whikloj Here's the changes you'd requested from [Pull request 147](https://github.com/Islandora/islandora_paged_content/pull/147)

strpos & available spelling, etc.

**Comment by rosiel** [comment]
Looks ok on a glance, but I'm not sure how to replicate the headers to test this. @Islandora/7-x-1-x-committers ?

**Comment by bondjimbond** [comment]
@GarrettArm Is there a JIRA ticket attached to this?

**Comment by GarrettArm** [comment]
I'm not aware of a jira ticket r/t this issue.

**Comment by bondjimbond** [comment]
@GarrettArm Can you please make one? (https://jira.duraspace.org) We need it for tracking reasons. Testing instructions (as in the template that appears when you make a pull request) will also be helpful. Thanks!

**Comment by GarrettArm** [comment]
Unfortunately, the pull request was five months ago.  These testing / jira requirements were not part of the agreement back then.  My employer has me working on other projects.  It's onerous to ask me to join your jira and setup an Islandora box + debugger just for this delinquent request.  If the code is good, please accept it; otherwise, I'll close it.

Obvious changes:
availible [changed to] available,
strstr [changed to] strpos,
"200" in line 1 of header == success 
[changed to]
"200"  in any line of header == success.  (E.g., proxies in front of the djatoka redirecting to a successful response are considered a success).

**Comment by bondjimbond** [comment]
JIRA ticket to match this PR: https://jira.duraspace.org/browse/ISLANDORA-2325

**Review by bondjimbond — APPROVED** [review]
Spelling improvements are good. The 200 header check seems to work.

**Comment by bondjimbond** [comment]
Will merge once a CLA is in.

**Comment by bondjimbond** [comment]
@manez Do we have a CLA for this org yet?

**Comment by DiegoPino** [comment]
@GarrettArm was working for Louisiana?

**Comment by manez** [comment]
We have a cCLA for LSU, so this is good to go.

**Comment by GarrettArm** [comment]
Yep -- work done at LSU.

**Pull request merged**
